install_config/install/advanced_install.adoc

```diff
@@ -306,6 +306,85 @@ allocate 10.128.0.0/23, 10.128.2.0/23, 10.128.4.0/23, and so on. This *cannot* b
 re-configured after deployment.
 |===
 
+[[advanced-install-configuring-global-proxy]]
+=== Configuring Global Proxy Options
+
+If your hosts require use of a HTTP or HTTPS proxy in order to connect to
+external hosts, there are many components that must be configured to use the
+proxy, including masters, nodes, Docker, and builds. In order to simplify this
+configuration, the following Ansible variables can be specified at a cluster
+level to apply these settings uniformly across your environment.
```

Comment:
Perhaps change this to `at a cluster or host level` and drop the next paragraph that mostly duplicates this sentence?